staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -153,16 +154,6 @@ type TLSConfig struct {
 type TracingConfiguration struct {
 	metav1.TypeMeta
 
-	// +optional
-	// Endpoint of the collector that's running on the control-plane node.
-	// The APIServer uses the egressType ControlPlane when sending data to the collector.
-	// The syntax is defined in https://github.com/grpc/grpc/blob/master/doc/naming.md.
-	// Defaults to the otlp grpc default, localhost:4317
-	// The connection is insecure, and does not currently support TLS.
-	Endpoint *string
-
-	// +optional
-	// SamplingRatePerMillion is the number of samples to collect per million spans.
-	// Defaults to 0.
-	SamplingRatePerMillion *int32
+	// Embed the component config tracing configuration struct
+	tracingapi.TracingConfiguration
```

Comment:
content of this type, just for reference

```go
	// Endpoint of the collector this component will report traces to.
	// The connection is insecure, and does not currently support TLS.
	// Recommended is unset, and endpoint is the otlp grpc default, localhost:4317.
	// +optional
	Endpoint *string `json:"endpoint,omitempty"`

	// SamplingRatePerMillion is the number of samples to collect per million spans.
	// Recommended is unset. If unset, sampler respects its parent span's sampling
	// rate, but otherwise never samples.
	// +optional
	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty"`
```